<ui version="4.0" >
 <class>InterestsInfoBase</class>
 <widget class="QWidget" name="InterestsInfoBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string></string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QTabWidget" name="TabWidget2" >
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>&amp;Interests</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>11</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QWidget" native="1" name="wndInterests" />
       </item>
       <item>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="0" column="0" >
          <widget class="QComboBox" name="cmbBg1" />
         </item>
         <item row="0" column="1" >
          <widget class="QLineEdit" name="edtBg1" />
         </item>
         <item row="3" column="1" >
          <widget class="QLineEdit" name="edtBg4" />
         </item>
         <item row="1" column="0" >
          <widget class="QComboBox" name="cmbBg2" />
         </item>
         <item row="3" column="0" >
          <widget class="QComboBox" name="cmbBg4" />
         </item>
         <item row="1" column="1" >
          <widget class="QLineEdit" name="edtBg2" />
         </item>
         <item row="2" column="0" >
          <widget class="QComboBox" name="cmbBg3" />
         </item>
         <item row="2" column="1" >
          <widget class="QLineEdit" name="edtBg3" />
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cmbBg1</tabstop>
  <tabstop>edtBg1</tabstop>
  <tabstop>cmbBg2</tabstop>
  <tabstop>edtBg2</tabstop>
  <tabstop>cmbBg3</tabstop>
  <tabstop>edtBg3</tabstop>
  <tabstop>cmbBg4</tabstop>
  <tabstop>edtBg4</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
